media-tv/plex-media-server/plex-media-server-1.23.4.4805.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1 systemd unpacker
+
+MY_PV="${PV}-186bae04e"
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+
+DESCRIPTION="Free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+S="${WORKDIR}"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="system-openssl"
+RESTRICT="mirror bindist"
+
+DEPEND="
+	acct-group/plex
+	acct-user/plex"
+RDEPEND="
+	${DEPEND}
+	system-openssl? ( dev-libs/openssl:0/1.1 )"
+
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/plexmediaserver/lib/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
+)
+
+src_install() {
+	# Remove Debian specific files
+	rm -r "usr/share/doc" || die
+
+	# Remove shipped openssl library
+	if use system-openssl; then
+		rm usr/lib/plexmediaserver/lib/libssl.so.1.1 || die
+	fi
```

Comment:
With this, I couldn't start plex-media-server (at least with start_pms command) - are you sure this is supposed to work?
```
# start_pms 
Error relocating /usr/lib64/libssl.so.1.1: ENGINE_load_ssl_client_cert: symbol not found
Error relocating /usr/lib64/libssl.so.1.1: ENGINE_finish: symbol not found
Error relocating /usr/lib64/libssl.so.1.1: ENGINE_init: symbol not found
Error relocating /usr/lib64/libssl.so.1.1: ENGINE_get_ssl_client_cert_function: symbol not found
```
Works if I install this with `-system-openssl`.

Reply:
Hm, I see libcrypto.so.1.1 is not being unbundled too, I thought it could be related (i.e. mismatch) but it also doesn't like the system's either:
```
Error relocating /usr/lib64/libcrypto.so.1.1: makecontext: symbol not found
Error relocating /usr/lib64/libcrypto.so.1.1: setcontext: symbol not found
```
As much as I dislike it, unsure if this IUSE can be supported at the moment.

Did this work before? I guess it's a good thing it's not enabled by default anyway.

Reply:
It did work with older version. It hasn't for the last two or three, though. I have removed the USE, and added a settings override file '/etc/default/plexmediaserver' so a user can keep changes through updates. Thanks for your time looking this over.